src/highs_bindings.cpp

```diff
@@ -1142,25 +1138,38 @@ PYBIND11_MODULE(_core, m) {
   m.attr("HIGHS_VERSION_PATCH") = HIGHS_VERSION_PATCH;
 
   // Submodules
-  py::module_ simplex_constants =
+  py::module_ simpc =
       m.def_submodule("simplex_constants", "Submodule for simplex constants");
 
-  py::enum_<SimplexStrategy>(simplex_constants, "SimplexStrategy")
-      .value("kSimplexStrategyMin", SimplexStrategy::kSimplexStrategyMin)
-      .value("kSimplexStrategyChoose", SimplexStrategy::kSimplexStrategyChoose)
-      .value("kSimplexStrategyDual", SimplexStrategy::kSimplexStrategyDual)
-      .value("kSimplexStrategyDualPlain",
-             SimplexStrategy::kSimplexStrategyDualPlain)
-      .value("kSimplexStrategyDualTasks",
-             SimplexStrategy::kSimplexStrategyDualTasks)
-      .value("kSimplexStrategyDualMulti",
-             SimplexStrategy::kSimplexStrategyDualMulti)
-      .value("kSimplexStrategyPrimal", SimplexStrategy::kSimplexStrategyPrimal)
-      .value("kSimplexStrategyMax", SimplexStrategy::kSimplexStrategyMax)
-      .value("kSimplexStrategyNum", SimplexStrategy::kSimplexStrategyNum)
+  py::enum_<SimplexStrategy>(simpc, "HighsSimplexStrategy")
+      .value("kMin", SimplexStrategy::kSimplexStrategyMin)
+      .value("kChoose", SimplexStrategy::kSimplexStrategyChoose)
+      .value("kDual", SimplexStrategy::kSimplexStrategyDual)
+      .value("kDualPlain", SimplexStrategy::kSimplexStrategyDualPlain)
+      .value("kDualTasks", SimplexStrategy::kSimplexStrategyDualTasks)
```

Comment:
As discussed also in the response, the naming is simply to represent how this looks in Python:

```python
HighsSimplexCrashStrategy.kMin
# or, earlier
SimplexCrashStrategy.kSimplexStrategyMin
```